aemedge/styles/styles.css

```diff
@@ -52,7 +52,8 @@
   --udexGridMargins: var(--udexGridXSMargins);
   --sapFontFamily: '72 Brand Variable', '72 Brand Variable Fallback', 'Arial',
     'Helvetica', 'sans-serif';
-  --hero-margin: var(--udexGridMargins);
+  --hero-margin--left: var(--udexGridMargins);
```

Comment:
I know `--hero-margin` was there before, but setting extra variables based on predefined udex variables is maybe not a good choice. `--udexGridMargins` can be used in hero.css directly.

Reply:
@mhaack --hero-margin is set to multiple values not just --udexGridMargins.